docs/en/middleware/zenoh.md

```diff
@@ -0,0 +1,194 @@
+
+```sh
+export RMW_IMPLEMENTATION=rmw_zenoh_cpp
+```
+
+Then start the Zenoh router using the command:
+
+```sh
+ros2 run rmw_zenoh_cpp rmw_zenohd
+```
+
+For more information about the Zenoh Router see the [rmw_zenoh](https://github.com/ros2/rmw_zenoh?tab=readme-ov-file#start-the-zenoh-router) documentation.
+
+## PX4 Zenoh-Pico Node Setup
+
+### PX4 Firmware
+
+Before setting up the Zenoh communication, first make sure that your firmware contains the driver that implements the [`zenoh` driver](../modules/modules_driver.html#zenoh), which provides the implementation of the _PX4 Zenoh-Pico Node_.
+
+You can check if the module is present on your board by searching for the key `CONFIG_MODULES_ZENOH=y` in your board's `default.px4board` KConfig file.
+For example, you can see that the module is present in `px4_fmu-v6xrt` build targets from [/boards/px4/fmu-v6xrt/default.px4board](https://github.com/PX4/PX4-Autopilot/blob/main/boards/px4/fmu-v6xrt/default.px4board#L91).
+If it is not present you can add this key to your board configuration and rebuild.
+
+The table below shows some of the PX4 targets that include this module (at time of writing).
+
+| PX4 Target             | Notes                                                                                       |
+| ---------------------- | ------------------------------------------------------------------------------------------- |
+| `px4_fmu-v6xrt`        | For [FMUv6X-RT](../flight_controller/nxp_mr_vmu_rt1176.md) (reference platform for testing) |
+| `nxp_tropic-community` |                                                                                             |
+| `nxp_mr-tropic`        |                                                                                             |
+| `nxp_mr-canhubk344`    |                                                                                             |
+| `px4_sitl_zenoh`       | Zenoh-enabled simulation build                                                              |
+| `px4_fmu-v6x_zenoh`    | Zenoh-enabled firmware for FMUv6X                                                           |
+
+Zenoh is not included in the default `px4_fmu-` targets for any firmware other than `px4_fmu-v6xrt` (`px4_sitl_zenoh` and `px4_fmu-v6x_zenoh` [are build variants](../dev_setup/building_px4.md#px4-make-build-targets)).
+
+::: tip
+You can check if Zenoh is present at runtime by using QGroundControl to [find the parameter](../advanced_config/parameters.md#finding-a-parameter) [ZENOH_ENABLE](../advanced_config/parameter_reference.md#ZENOH_ENABLE).
+If present, the module is installed.
+:::
+
+### Enable Zenoh on PX4 Startup
+
+Set the [ZENOH_ENABLE](../advanced_config/parameter_reference.md#ZENOH_ENABLE) parameter to `1` to enable Zenoh on PX4 startup.
+
+### Configure Zenoh Network
+
+Set up PX4 to connect to the companion computer running `zenohd`.
+
+PX4's default IP address of the Zenoh daemon host is `192.168.1.104`.
```

Comment:
I assume this is the default.

Reply:
This is just the IP address of the companion right? If so the default is 10.41.10.2 according to https://docs.px4.io/main/en/advanced_config/ethernet_setup#setting-up-the-ethernet-network

Note, this changed in v1.16. The release note says:

> Ethernet static IP changed from 192.168.0.3 to 10.41.10.2 to avoid common network conflicts ([PX4-Autopilot#22517](https://github.com/PX4/PX4-Autopilot/pull/22517))

Reply:
Right, we went for `10.41.10.1` for the companion and `10.41.10.2` for the fmu

Reply:
Thanks, @hamishwillee I've modified the default behavior to `10.41.10.1` and updated the docs to reflect this.

Reply:
Thanks!